streams/src/test/java/org/apache/kafka/streams/integration/EosIntegrationTest.java

```diff
@@ -176,13 +176,13 @@ public void createTopics() throws Exception {
     }
 
     @ParameterizedTest
-    @ValueSource(strings = {StreamsConfig.AT_LEAST_ONCE, StreamsConfig.EXACTLY_ONCE, StreamsConfig.EXACTLY_ONCE_V2})
+    @ValueSource(strings = {StreamsConfig.AT_LEAST_ONCE, StreamsConfig.EXACTLY_ONCE_V2})
```

Comment:
Why do we run this "eos test" with at-least-once ?

Reply:
It seems the at-least-once was added by #11283

Maybe we can have a follow-up to rename this class or move the at-least-once tests out of this class.

Reply:
Seems this is a mess -- the original PR actually also added "early returns" for some test... Guess we need to do more follow up cleanup.

Seem one root cause if, that we have a centralized parameter originally, but now a per-test one.